pkg/importer/dataStream.go

```diff
@@ -98,15 +99,15 @@ const httpClientTimeout = time.Minute * 60
 
 // NewDataStream returns a DataStream object after validating the endpoint and constructing the reader/closer chain.
 // Note: the caller must close the `Readers` in reverse order. See Close().
-func NewDataStream(endpt, accKey, secKey string) (*DataStream, error) {
-	return newDataStream(endpt, accKey, secKey, nil)
+func NewDataStream(endpt, accKey, secKey, content string) (*DataStream, error) {
+	return newDataStream(endpt, accKey, secKey, content, nil)
 }
 
 func newDataStreamFromStream(stream io.ReadCloser) (*DataStream, error) {
-	return newDataStream("stream://data", "", "", stream)
+	return newDataStream("stream://data", "", "", common.ImporterContentKubvirt, stream)
```

Comment:
Kubevirt

Reply:
Irrelevant anymore after rebasing on top of #520.